examples/demo.css

```diff
@@ -69,14 +69,24 @@ body > .container-fluid {
   min-width:300px;
 }
 
-.hover {
-  background: #eee;
-}
-
 .dropzone input {
   max-width: 100%;
 }
 
+.import-image-zone {
+  margin: 10px auto 30px auto;
+  max-width: 250px;
+  min-width: 230px;
+}
+
+.import-image-zone input {
+  max-width: fit-content;
```

Comment:
please change this as well to max-width: 100%;

Reply:
If you want the box to look like this, I can do that :)

![prob](https://user-images.githubusercontent.com/35997844/70376494-cc3a6300-1909-11ea-8d0f-3f2f5ccaba2f.png)

Reply:
That's why .import-image-zone input needs to be:
```css
.import-image-zone input {
    max-width: fit-content;
}
```
or
```css
.import-image-zone input {
    max-width: 101%;
}
```

Reply:
That 1% less(ie. 100%) squishes text. I suggest that we keep it at fit-content.

Reply:
it won' squash if you keep max-width :250 px in .import-image-zone and 100% .import-image-zone input .I have already tried it locally.Can you also give one more try.

Reply:
.import-image-zone {
  margin: 10px auto 30px auto;
  max-width: 250px;
  min-width: 230px;
}

.import-image-zone input {
  max-width: 100%;
}